student.rb

```diff
@@ -1,16 +1,19 @@
-class Student
+require_relative 'teacher'
+
+class Student < Teacher
```

Comment:
students don't inherit from a teacher. A child inherits characteristics from their parent, but a student is a different type of person than a teacher. Maybe there is a new super class - Person - that they can both inherit from